include/sys/multilist.h

```diff
@@ -74,6 +74,8 @@ struct multilist {
 void multilist_destroy(multilist_t *);
 void multilist_create(multilist_t *, size_t, size_t,
     multilist_sublist_index_func_t *);
+void multilist_create_impl(multilist_t *, size_t, size_t,
+    unsigned int, multilist_sublist_index_func_t *);
```

Comment:
Should this `*_impl` function be part of a public header?  My understanding was that the style was to keep `*_impl` functions private as much as possible and provide wrappers for this.  This way the *_impl function prototype can be easily changed without worrying about changing all the callers.